specreduce/tests/test_wavelength_calibration.py

```diff
@@ -109,3 +109,25 @@ def test_fit_residuals_access(spec1d):
                                    line_wavelengths=w)
     test.residuals
     test.wcs
+
+
+def test_unsorted_pixels_wavelengths(spec1d):
```

Comment:
can we also add a test for the inverted case to make sure that does not pass?  Perhaps `test_linear_from_list` can just have a second case that does `WavelengthCalibration1D(spec1d, line_pixels=centers[::-1], line_wavelengths=w)`.